### PR TITLE
Import Excel in admin

### PR DESCRIPTION
**Pull request by kams2391**

### CODE REVIEW

File: app/Imports/ImportCategory.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+					   'instruction_of_use' => $row[53]??'',
+				   ]);
+				    
+				   $product_id=$product['id'];
+			   }
+		   $cat_arr=explode(',',$row[5]);
+		   
+		   if(count($cat_arr)>0){
+		   //Add multiple category
+		   foreach($cat_arr as $k=>$v){
```
Comment (upworksanjeev) [inline app/Imports/ImportCategory.php:43]: @kams2391 Please check if we can use Heading row here: https://docs.laravel-excel.com/3.1/imports/heading-row.html

File: app/Imports/ImportCategory.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+			   	$catprod=CategoryProduct::where('category_id',$cat_id)->where('product_id',$product_id)->first();		   
+			   //Add category_product relation
+			    if(!isset($catprod['id'])){
+					$catprod=CategoryProduct::create(['category_id' => $cat_id,'product_id' => $product_id]);
+				}
+			}
+		   }
+		   $j=1;
+		   /* Add attributes of product */
+		   for($i=6;$i<52;$i++){
```
Comment (upworksanjeev) [inline app/Imports/ImportCategory.php:67]: @kams2391 Mam ye loop hum kyo cla rhe hai.

File: app/Imports/ImportCategory.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+					if(!isset($prod_attr['id'])){				
+						$prod_attr=ProductAttribute::create(['prod_id' => $product_id,'attr_id' => $att_id]);
+					}
+			}
+			$j++;
+		   }		  
+		  return $product;
+		}else{
+			/* Add attributes category */
+		   for($i=6;$i<52;$i++){
```
Comment (upworksanjeev) [inline app/Imports/ImportCategory.php:86]: same for this one

Comment (kams2391) [reply app/Imports/ImportCategory.php]: removed

File: app/Imports/ImportCategory.php (hunk 2 above)
Comment (kams2391) [reply app/Imports/ImportCategory.php]: removed.. now working with heading names

File: app/Imports/ImportCategory.php (hunk 1 above)
Comment (kams2391) [reply app/Imports/ImportCategory.php]: It is done.

**Review by upworksanjeev — APPROVED** [review]

**Pull request merged**